### PR TITLE
Improve ActionText::FixtureSet documentation

### PR DESCRIPTION
**Pull request by seanpdoyle**
Support for Action Text attachments in fixtures was added by [76b33aa][] and
released as part of [6.1.1][], but has not yet been documented.

This commit documents the `ActionText::FixtureSet` for the API
documentation, and mentions it in the Rails Guides pages.

[76b33aa]: https://github.com/rails/rails/commit/76b33aa3d102ad4ff3f766ac2d3a711d73aaee82
[6.1.1]: https://github.com/rails/rails/releases/tag/v6.1.1

### CODE REVIEW

File: actiontext/lib/action_text/fixture_set.rb [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+    #
+    #   # tests/fixtures/articles.yml
+    #   second:
+    #     title: Another Article
+    #
+    #   # tests/fixtures/action_text/rich_texts.yml
+    #   second_content:
+    #     record: second (Article)
+    #     name: content
+    #     body: <div>Hello, <%= ActionText::FixtureSet.attachment("artcles", :first) %></div>
```
Comment (seanpdoyle) [inline actiontext/lib/action_text/fixture_set.rb:54]: @georgeclaghorn after having a chance to reflect, how do we feel about the separation of `ActionText::FixtureSet` from `ActiveRecord::FixtureSet`?

**Comment by sedubois** [comment]
Would it be possible to have a clear header in the guide titled something like "Testing Action Text"? Currently searching the guide with the text "action text" yields no result.

**Comment by seanpdoyle** [comment]
> Would it be possible to have a clear header in the guide titled something like "Testing Action Text"? Currently searching the guide with the text "action text" yields no result.

@sedubois the docs changes are so brief that I can't think of a way to squeeze a title in. Would it be ok if I inject the words "Action Text" into the existing changes instead?

**Comment by sedubois** [comment]
Sure, that would already be an improvement 👍

Ultimately it would be helpful to have sections "Testing Active Storage" and "Testing Action Text", in addition to the existing "Testing Your Mailers", "Testing Jobs", "Testing Action Cable", etc.

File: actiontext/lib/action_text/fixture_set.rb [hunk 2]
```diff
@@ -1,7 +1,57 @@
 # frozen_string_literal: true
 
 module ActionText
+  # \Fixtures are a way of organizing data that you want to test against; in
+  # short, sample data.
+  #
+  # To learn more about fixtures, read the ActiveRecord::FixtureSet
```
Comment (MatheusRich) [inline actiontext/lib/action_text/fixture_set.rb:7]: Can we add a link here?

File: actiontext/lib/action_text/fixture_set.rb [hunk 3]
```diff
@@ -1,7 +1,57 @@
 # frozen_string_literal: true
 
 module ActionText
+  # \Fixtures are a way of organizing data that you want to test against; in
```
Comment (MatheusRich) [inline actiontext/lib/action_text/fixture_set.rb:4]: Is this intentional or a typo?

```suggestion
  # Fixtures are a way of organizing data that you want to test against; in
```

File: actiontext/lib/action_text/fixture_set.rb (hunk 2 above)
Comment (seanpdoyle) [reply actiontext/lib/action_text/fixture_set.rb:7]: That sounds great! What's the rdoc syntax for that?

File: actiontext/lib/action_text/fixture_set.rb (hunk 3 above)
Comment (seanpdoyle) [reply actiontext/lib/action_text/fixture_set.rb:4]: It's lifted from the ActiveRecord::FixtureSet class doc:

https://github.com/rails/rails/blob/76c834bcb67d947e22a126071d12d98cda511faf/activerecord/lib/active_record/fixtures.rb#L18

I'd appreciate any and all rdoc syntax guidance!

File: actiontext/lib/action_text/fixture_set.rb (hunk 2 above)
Comment (MatheusRich) [reply actiontext/lib/action_text/fixture_set.rb:7]: http://docs.seattlerb.org/rdoc/RDoc/Markup.html#class-RDoc::Markup-label-Links

File: actiontext/lib/action_text/fixture_set.rb (hunk 1 above)
Comment (dhh) [reply actiontext/lib/action_text/fixture_set.rb:53]: I think it's fine and good. Active Record shouldn't know about the setup required for fixtures in Action Text.

**Pull request merged**